.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 80,
+  "useTabs": false,
+  "tabWidth": 2
+}
```

Comment:
Those 3 already are the default values, but I wouldn't mind having trailing commas here if you prefer

Reply:
Added `trailingComma` to make git diffs smaller and moving lines easier :)

Reply:
But you can remove the other 3 lines as they are already the default values: https://prettier.io/docs/en/options.html